ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml

```diff
@@ -123,7 +123,7 @@ tests:
     workflow: openshift-qe-installer-aws
   timeout: 5h0m0s
 - as: data-path-9nodes
-  cron: 0 10 1,15 * *
+  cron: 0 10 18 * *
```

Comment:
18 isn't a valid day . Please stick with 1,8,15,22 for data-path tests.

Reply:
Picked 8.